.github/workflows/build.yml

```diff
@@ -38,10 +38,17 @@ jobs:
       run: bundle exec hatchet ci:setup
     - name: Run test suite
       run: PARALLEL_SPLIT_TEST_PROCESSES=25 bundle exec parallel_split_test spec/
```

Comment:
I think this might be more readable if the `PARALLEL_SPLIT_TEST_PROCESSES` env var was moved under `env` along with the other env vars?

Reply:
Done